GSoC-interest.md

```diff
@@ -19,6 +19,8 @@ GSoC Mentors
 | Name | Email | Idea | Repo | Project Proposal |
 | --- | --- | --- | --- | --- |
 | Georg Link | glink at unomaha.edu | Prototype New CHAOSS Metrics | [GeorgLink/governance](https://github.com/GeorgLink/governance) | [none](https://wiki.linuxfoundation.org/chaoss/gsoc-ideas) |
+| Keanu Nichols | nichols.keanu9@gmail.com | 	Reporting of CHAOSS Metrics | [kmn5409/chaoss-microtasks]
+(https://github.com/kmn5409/chaoss-microtasks) | To-Do |
```

Comment:
@kmn5409, You must combine the lines and please add them to the bottom of the list.